<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoansDialog</class>
 <widget class="QDialog" name="LoansDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Loans</string>
  </property>

  <widget class="QListWidget" name="loansListWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>360</width>
     <height>180</height>
    </rect>
   </property>
  </widget>

  <widget class="QDoubleSpinBox" name="amountSpinBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="prefix">
    <string>â‚¬</string>
   </property>
   <property name="maximum">
    <double>1000000.0</double>
   </property>
  </widget>

  <widget class="QDoubleSpinBox" name="interestSpinBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="suffix">
    <string>%</string>
   </property>
   <property name="maximum">
    <double>100.0</double>
   </property>
   <property name="value">
    <double>5.0</double>
   </property>
  </widget>

  <widget class="QPushButton" name="requestLoanButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>210</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Request Loan</string>
   </property>
  </widget>

  <widget class="QPushButton" name="payLoanButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Pay Selected Loan</string>
   </property>
  </widget>

  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>

 </widget>
 <resources/>
 <connections/>
</ui>
